src/styles/_signup.scss

```diff
@@ -1,6 +1,6 @@
-h1 {
-    margin-bottom: 10%;
-}
+// h1 {
```

Comment:
Good job, but you need to remove this commented code

Reply:
done